packages/steveo/src/producers/kafka.ts

```diff
@@ -63,12 +63,23 @@ class KafkaProducer
     });
   }
 
-  getPayload = <T>(payload: T) => {
-    const context = createMessageMetadata(payload);
+  getPayload = <T>(
+    payload: T,
+    _topic: string,
```

Comment:
QQ: is there any use of this args atm or are we trying to make it the same signature with SQS' getPayload?

Reply:
Typescript complained when I tried to add the context params because the signature didn't match the IProduce::getPayload contract. That's why I need to add topic and key, but they are not used, hence the `_` prefix.

Reply:
@lmagalhaes ah my bad, just saw the changes in IProducer, thought getPayload is a private method, Yup make sense 👍